src/views/UrbitOS/Home.js

```diff
@@ -0,0 +1,188 @@
+import { usePointCache } from 'store/pointCache';
+
+import useCurrentPointName from 'lib/useCurrentPointName';
+import useKeyfileGenerator from 'lib/useKeyfileGenerator';
+import * as need from 'lib/need';
+
+import {
+  OutButton,
+  ForwardButton,
+  BootUrbitOSButton,
+} from 'components/Buttons';
+import CopyButton from 'components/CopyButton';
+import NetworkingKeys from 'components/NetworkingKeys';
+
+import BridgeForm from 'form/BridgeForm';
+import { useLocalRouter } from 'lib/LocalRouter';
+
+export default function UrbitOSHome() {
+  const { pointCursor } = usePointCursor();
+  const { getDetails } = usePointCache();
+
+  const { push, names } = useLocalRouter();
+
+  const point = need.point(pointCursor);
+  const details = need.details(getDetails(point));
+
+  const sponsor = ob.patp(details.sponsor);
+
+  const { code, notice } = useKeyfileGenerator();
+
+  const [showKeys, setShowKeys] = useState(false);
+
+  const showSponsor = azimuth.getPointSize(point) !== azimuth.PointSize.Galaxy;
+  const toggleShowKeys = useCallback(() => setShowKeys(!showKeys), [
+    setShowKeys,
+    showKeys,
+  ]);
+
+  const goNetworkingKeys = useCallback(() => push(names.NETWORKING_KEYS), [
+    names,
+    push,
+  ]);
+
+  const goChangeSponsor = useCallback(() => push(names.CHANGE_SPONSOR), [
+    push,
+    names,
+  ]);
+  return (
+    <>
+      <Grid>
+        <Grid.Item full className="mv7">
+          Urbit OS
+        </Grid.Item>
+        <Grid.Divider />
+        <Grid.Item full as={BootUrbitOSButton} />
+      </Grid>
+
+      <Grid>
+        <Grid.Item full className="mv7 f5">
+          Network
+        </Grid.Item>
+        <Grid.Divider />
+        {showSponsor && (
+          <>
+            <Grid.Item
+              full
+              as={ForwardButton}
+              detail="A sponsor finds new peers in your network"
+              accessory={<u>Change</u>}
+              onClick={goChangeSponsor}>
+              <span className="mono">{sponsor}</span>
+              <span className="f7 bg-black white p1 ml2 r4">SPONSOR</span>
+            </Grid.Item>
+            <Grid.Divider />
+          </>
+        )}
+        <Grid.Item
+          full
+          as={ForwardButton}
+          accessory={code && <CopyButton text={code} />}
+          detail={code || notice}
+          disabled={!code}
+          detailClassName="mono">
+          Login Code
+        </Grid.Item>
+        <Grid.Divider />
+        <Grid.Item full as={ForwardButton} onClick={goNetworkingKeys}>
+          Reset Networking Keys
+        </Grid.Item>
+        <Grid.Divider />
+        <Grid.Item
+          full
+          as={ForwardButton}
+          accessory={showKeys ? '▲' : '▼'}
+          onClick={toggleShowKeys}>
+          View Networking Keys
+        </Grid.Item>
+        {showKeys && <Grid.Item full as={NetworkingKeys} point={point} />}
+      </Grid>
+    </>
+  );
+}
+
+// eslint-disable-next-line
+function Hosting() {
```

Comment:
All of the hosting stuff here seems like stubs, and isn't used/displayed anywhere. We should probably leave the code out for now, especially for this PR.